src/ci/dotnet-fuzzing-tools.sh

```diff
@@ -34,7 +34,7 @@ popd
 
 # Build SharpFuzz and our dynamic loader harness for `libfuzzer-dotnet`.
 pushd src/agent/LibFuzzerDotnetLoader
-dotnet publish . -c Release -o $ARTIFACTS/LibFuzzerDotnetLoader --sc -r linux-x64 -p:PublishSingleFile=true
+dotnet publish . -c Release -o $ARTIFACTS/LibFuzzerDotnetLoader --sc -r linux-x64
```

Comment:
Suggest using the full name of this flag as isn't as clear as the arguments which take a value are:
```suggestion
dotnet publish . -c Release -o $ARTIFACTS/LibFuzzerDotnetLoader --self-contained -r linux-x64
```